src/Http/Authentication.Abstractions/src/Microsoft.AspNetCore.Authentication.Abstractions.csproj

```diff
@@ -11,7 +11,6 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
-    <Reference Include="Microsoft.Extensions.Logging.Abstractions" />
```

Comment:
This references are brought in from the transport package. Having / removing it does nothing, but it seemed unnecessary.